ibm/service/power/data_source_ibm_pi_instances.go

```diff
@@ -273,6 +273,7 @@ func flattenPvmInstanceNetworks(list []*models.PVMInstanceNetwork) (networks []m
 			p := make(map[string]interface{})
 			p[Attr_ExternalIP] = pvmip.ExternalIP
 			p[Attr_IP] = pvmip.IPAddress
+			p[Attr_Macaddress] = pvmip.MacAddress
```

Comment:
If we are using `Attr_MacAddress` everywhere in this PR's new code, why are we adding another attr? that too without any documentation?

Reply:
Din't realize the doc for instances was already in place.